### ISSUE TITLE
Increase Prometheus Cluster monitor proxy NGINX buffers

### ISSUE THREAD

**Issue by cloudnautique** [rancher/rancher#19689]
Rancher/rancher:v2.2.2

It appears we are exceeding the buffers through the proxy:

```
2019/04/17 22:49:57 [warn] 7#7: *10169 an upstream response is buffered to a temporary file /tmp/temp_dir/0000000033 while reading upstream, client: 10.32.0.0, server: , request: "GET /api/datasources/proxy/1/api/v1/query_range?query=sum%20(container_memory_working_set_bytes%7Bid%3D%22%2F%22%7D)%20by%20(node)%20%2F%20sum%20(machine_memory_bytes)%20by%20(node)%20*%20100&start=1555537785&end=1555541400&step=15 HTTP/1.1", upstream: "http://127.0.0.1:3000/api/datasources/proxy/1/api/v1/query_range?query=sum%20(container_memory_working_set_bytes%7Bid%3D%22%2F%22%7D)%20by%20(node)%20%2F%20sum%20(machine_memory_bytes)%20by%20(node)%20*%20100&start=1555537785&end=1555541400&step=15", host: "bill-loadtest.rancher.io", referrer: "https://bill-loadtest.rancher.io/k8s/clusters/c-tht6z/api/v1/namespaces/cattle-prometheus/services/http:access-grafana:80/proxy/d/icjpCppik/cluster?refresh=1m&orgId=1&from=now-1h&to=now"
```

This might be just increasing:
```
proxy_buffers 16 16k;  
proxy_buffer_size 16k;
```

This cluster has 256 nodes and a few of the grafana dashboards open. 
It looks like this dataset is ~ 2MB
![Screen Shot 2019-04-17 at 3 56 35 PM](https://user-images.githubusercontent.com/1989084/56326074-72421e80-6129-11e9-99dc-12a90d2613e1.png)

**Comment by loganhz** [comment]
Chart PR merged 

Waiting for https://github.com/rancher/rancher/pull/19766 https://github.com/rancher/types/pull/801 https://github.com/rancher/system-charts/pull/37 merged and https://github.com/rancher/rancher/pull/19766 https://github.com/rancher/types/pull/801 backport to 2.2. branch.

**Comment by cloudnautique** [comment]
@thxCode if we are dealing with 2MB data sets, I would have expected the proxy_buffer be something more like `128 16k` or `256 16k`? or even some larger number by 8k? 

I had just copied the settings that possibly need to be adjusted, and not done any analysis on what the settings should be. Do you have some numbers that you did to arrive at this update?

**Comment by cloudnautique** [comment]
Actually discussing internally, we should disable proxy_buffer for this location.

**Comment by thxCode** [comment]
Ok, we will adjust this today. Thanks.

**Comment by jiaqiluo** [comment]
The bug fix is validated on master 5/31 ecbec960
Steps:
- add a DO cluster
- enable cluster monitoring
- execute Shell on the container `grafana-proxy` in `Workload: grafana-cluster-monitoring` to check its `nginx.conf` file
- execute Shell on the container `prometheus-proxy` in `Workload: prometheus-cluster-monitoring` to check its `nginx-conf.tmpl` file 
- confirm to see the following line in both files
```
 proxy_buffering             off;
```
